tests/api/auth/test_basic.py

```diff
@@ -217,14 +220,17 @@ def test_get_whoami_correct_credentials(basic_auth_test_client, fs):
 
     auth_file_path = settings.APP_DIR / "auth.json"
     fs.create_file(auth_file_path, contents=STORED_CREDENTIALS)
-    get_authenticated_user.cache_clear()
+    get_basic_auth_user.cache_clear()
 
     response = basic_auth_test_client.get(
         "/whoami", headers={"Authorization": f"Basic {credentials}"}
     )
 
     assert response.status_code == 200
-    assert response.json() == {
```

Comment:
Why did you change and split this assertion ?

Reply:
Following recent remarks on the use of sets vs lists, I decided to use sets to store scopes, which means we cannot assume an order in the scopes we retrieve.